lib/flipper/adapters/memcached.rb

```diff
@@ -0,0 +1,74 @@
+require 'dalli'
+
+module Flipper
+  module Adapters
+    class Memcached
```

Comment:
Since this is dalli and some apps use the memcached gem, I'd prefer to call this adapter dalli I think. Helps avoid confusing and allows for a memcached adapter down the road.